packages/validator/src/services/doppelgangerService.ts

```diff
@@ -51,10 +51,7 @@ export class DoppelgangerService {
       metrics.doppelganger.statusCount.addCollect(() => this.onScrapeMetrics(metrics));
     }
 
-    this.logger.info("Doppelganger protection enabled", {
-      currentEpoch: this.clock.currentEpoch,
-      detectionEpochs: DEFAULT_REMAINING_DETECTION_EPOCHS,
-    });
+    this.logger.info("Doppelganger protection enabled", {detectionEpochs: DEFAULT_REMAINING_DETECTION_EPOCHS});
```

Comment:
I added epoch to the log in https://github.com/ChainSafe/lodestar/pull/5987 but noticed we only want the epoch information when registering a validator. Better to only log doppelganger related configuration here.